docs/internals/frontends.rst

```diff
@@ -152,8 +245,222 @@ username
 comment
     Archive comment, if any
 
+Example of a simple archive listing (``borg list --last 1 --json``)::
+
+    {
+        "archives": [
+            {
+                "archive": "2017-02-27T21:21:51",
```

Comment:
this might be a bit confusing if you archive name just happens to be a datetime string.
maybe include example host and/or dataset name to make it clearer.